server/ansible/roles/pbench-server-enable/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+# Example: /var/www/html/pbench-results-host-info.versioned/pbench-results-host-info.URL002
+#          ^-----------^ ^-------------------pbench_host_info_prefix-------------------^
+#                ^
+#     httpd_document_root_dir
+- name: "set up links to the active files"
```

Comment:
This looks almost identical to the "disable" role.  Could we remove the duplication here?

Reply:
Thinking about this...

Reply:
Fixed.